lib/group_site_manager.py

```diff
@@ -154,6 +156,44 @@ def _load_page(self, driver, url) -> None:
     driver.get(url)
     time.sleep(3)
 
+  def _upload_bfmr(self, numbers) -> None:
+    group_config = self.config['groups']['bfmr']
+    driver = self.driver_creator.new()
+    try:
+      # load the login page first
+      self._load_page(driver, "https://buyformeretail.com/login")
+      driver.find_element_by_id("loginEmail").send_keys(group_config['username'])
+      driver.find_element_by_id("loginPassword").send_keys(
+          group_config['password'])
+      driver.find_element_by_xpath("//button[@type='submit']").click()
+
+      time.sleep(2)
+
+      # hope there's a button to submit tracking numbers -- it doesn't matter which one
+      submit_button = driver.find_element_by_xpath(
+          "//button[text() = \"Submit tracking #'s\"]")
+      submit_button.click()
+
+      time.sleep(2)
+
+      modal = driver.find_element_by_class_name("modal-body")
+      form = modal.find_element_by_tag_name("form")
+
+      # put in the tracking, click the add-new-number button
+      for i in range(len(numbers)):
+        number = numbers[i]
+        input_elem = form.find_elements_by_xpath("//input[@appvalidinput]")[i]
+        input_elem.send_keys(number)
+        form.find_element_by_xpath(
+            "//span[text() = 'Add Tracking number']").click()
+
+      # Submit twice. If there are dupes, it removes them after the first time and submits only the new ones the second time
+      form.find_element_by_xpath("//button[text() = 'Submit']").click()
+      time.sleep(1)
+      form.find_element_by_xpath("//button[text() = 'Submit']").click()
```

Comment:
There's no good way to check the return value of the first one rather than always clicking it twice?

Reply:
yeah that's probably true -- unfortunately this only gives that error when the only trackings we have are new ones, which is difficult to test...

Reply:
aight this is a bit better, hopefully 

haven't had a chance to test it yet with purely-new trackings